pagopa/io-app
# feat(IT Wallet): [SIW-1473] Add credential trustmark

## Short description
This PR adds the trustmark on the driving license, health card and disability card. A QR code containing a url to the verifier web application is generated from the credential.

> [!NOTE]  
> The verifier url is currently mocked.

## List of changes proposed in this pull request
- Created `ItwCredentialTrustmark` component
- Added util to generate the QR code verification url

## How to test
Open the detail of one of the aforementioned credentials. The trustmark button should be visible and should open a bottom sheet with a QR code.

When scanned, it should contain a link with the following parameters: `data_hash`, `signature` and `kid`.
